keyboards/alectardy98/alpinenoon/keymaps/vial_alec/readme.md

```diff
@@ -0,0 +1 @@
+# The preferd alec vial keymap for Alpine Noon
```

Comment:
All vial content should be removed from a QMK PR as referenced in the PR guideline.